public/pages/CreateTrigger/components/TriggerQuery/TriggerQuery.js

```diff
@@ -113,17 +112,22 @@ const TriggerQuery = ({
             )}
           />
         </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <EuiFormRow label="Trigger condition response:" fullWidth>
+            <EuiCodeEditor
+              theme={isDarkMode ? 'sense-dark' : 'github'}
```

Comment:
I think EUI can handle the theme by default. Can you confirm if it works for all browsers if remove this line?

Reply:
CodeEditor doesn't handle it so we need to handle it ourself.